src/cmd/terraform.go

```diff
@@ -279,7 +279,12 @@ func exportServices(c *opslevel.Client, shell *os.File, directory string) {
 		if len(aliases) > 0 {
 			aliases = fmt.Sprintf("aliases = [\"%s\"]", aliases)
 		}
-		tags := flattenTags(service.Tags.Nodes)
+		serviceTags, err := service.GetTags(c, nil)
+		if err != nil {
```

Comment:
I understand why we need a `// TODO` block, but I think `cobra.CheckErr(err)` would be better than the `if` block.

Reply:
ya especially since we do that 8 lines up.  The end user for the most part is trying to dump out their terraform configuration so they'll see this error (it won't be in an automated system) so they will beable to react to the failure.

Reply:
Ah, both good calls 👍  Putting this in

Reply:
We can get rid of the `if err != nil` block altogether since we're using `cobra.CheckErr()`